### PR TITLE
Add Vineyard Operator Documention

### PR DESCRIPTION
**Pull request by dashanji**
The pr gives a detailed doc about Vineyard Operator.

Related to https://github.com/v6d-io/v6d/issues/1031

### CODE REVIEW

File: docs/notes/vineyard-operator.rst [hunk 1]
```diff
@@ -0,0 +1,1058 @@
...
+  NAMESPACE         NAME                ID                  NAME   SIGNATURE           TYPENAME              INSTANCE   HOSTNAME
+  vineyard-system   o001d2dc18d72a47e   o001d2dc18d72a47e          s001d2dc18d729ab6   vineyard::DataFrame   2          kind-worker2
+
+The whole workflow can be found in `dask repartition e2e test <https://github.com/v6d-io/v6d/blob/main/k8s/test/e2e/repartition/dask-repartition-e2e.yaml>`_.
+What's more, please refer the `repartition directory <https://github.com/v6d-io/v6d/tree/main/k8s/test/e2e/repartition>`_ to get more details.
+
+Other Repartition Demo
+""""""""""""""""""""""
+
+To be done.
```
Comment (sighingnow) [inline docs/notes/vineyard-operator.rst:1052]: Just delete this subsection if no content currently.

File: docs/notes/vineyard-operator.rst [hunk 2]
```diff
@@ -0,0 +1,1058 @@
...
+
+End
+---
+
+Just have a try, and you will find it easy to use. If you have any questions, 
+please feel free to contact us.
```
Comment (sighingnow) [inline docs/notes/vineyard-operator.rst:1058]: No need for this.

**Comment by dashanji** [comment]
Thanks for the detailed review, all comments done, please take another look at this.

File: README.rst [hunk 3]
```diff
@@ -317,6 +273,7 @@ third-party libraries may not have the same license as vineyard.
 .. _Installation: https://github.com/v6d-io/v6d/blob/main/docs/notes/install.rst
 .. _Contributing: https://github.com/v6d-io/v6d/blob/main/CONTRIBUTING.rst
 .. _Getting Started: https://v6d.io/notes/getting-started.html
+.. _Vineyard Operator: https://github.com/dashanji/v6d/blob/add-vineyard-operator-doc/docs/notes/vineyard-operator.rst
```
Comment (sighingnow) [inline README.rst]: Don't link to your own fork.

File: charts/vineyard-operator/README.md [hunk 4]
```diff
@@ -29,6 +29,23 @@ helm uninstall vineyard-operator
...
+| service.type       | string | "ClusterIP"                                                               | The type of the service.                                                                                                   |
+| service.port       | int    | 9600                                                                      | The internal port of vineyard operator service                                                                             |
+| resources          | object | {limits: {cpu: 500m, memory:500Mi}},{requests: {cpu: 500m, memory:500Mi}} | The limits and requests of vineyard operator.                                                                              |
+| tolerations        | object | {}                                                                        | Tolerations allow the scheduler to schedule pods with matching taints                                                      |
+| affinity           | object | {}                                                                        | Affinity enables the scheduler to place a pod either on a group of nodes or a pod relative to the placement of other pods. |
+
```
Comment (sighingnow) [inline charts/vineyard-operator/README.md:48]: Could you please make this table less wider as well?

File: README.rst [hunk 5]
```diff
@@ -33,11 +33,7 @@ Table of Contents
   * `I/O Drivers <#drivers>`_
 
 * `Getting started with vineyard <#try-vineyard>`_
-* `Vineyard on Kubernetes <#integrate-with-kubernetes>`_
-
-  * `Deploying on Kubernetes <#deployment-on-kubernetes>`_
-  * `Deploying using Helm <#deployment-with-helm>`_
-
+* `Integrate with kubernetes <#integrate-with-kubernetes>`_
```
Comment (sighingnow) [inline README.rst:36]: Rename to "Deploying on Kubernetes".

**Review by sighingnow — APPROVED** [review]
LGTM now. Just some minor comments.

File: README.rst (hunk 3 above)
Comment (dashanji) [reply README.rst]: Yeah, The link is only for testing CI, I think the next commit will fail as the link of vineyard is deadline.

File: charts/vineyard-operator/README.md (hunk 4 above)
Comment (dashanji) [reply charts/vineyard-operator/README.md:48]: Get it.

**Pull request merged**